src/core/etl/src/Flow/ETL/Row/Factory/NativeEntryFactory.php

```diff
@@ -289,6 +303,11 @@ private function isJson(string $string) : bool
         }
     }
 
+    private function isUuid(string $string) : bool
+    {
+        return 0 !== \preg_match('/\A[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}\z/ms', $string);
```

Comment:
you came up with this regex or found it somewhere? Is it the same as symfony uid component is using?

Reply:
It's taken from Ramsey library actually, I was not sure if we need to place a copy-right for it or not.

Reply:
Not sure from where Ramsey lib took it but I would extract it to const and put a reference that it was taken from Ramsey lib with credits